biz.aQute.bnd/src/aQute/bnd/main/bnd.java

```diff
@@ -4583,4 +4585,63 @@ private File toFile(Container c) {
 		}
 		return c.getFile();
 	}
+
+	/**
+	 * Install the build tool
+	 */
+
+	interface BuildtoolOptions extends workspaceOptions {
+		boolean force();
+	}
+
+	public void _buildtool(BuildtoolOptions options) throws Exception {
+
+		Workspace ws = getWorkspace(options.workspace());
+		Parameters buildtool = new Parameters(ws.mergeProperties("-buildtool"));
+
+		for( Map.Entry<String, Attrs> entry : buildtool.entrySet()) {
+			String key = Processor.removeDuplicateMarker(entry.getKey());
+			Attrs attrs = entry.getValue();
+
+			String url = getUrl(key, attrs);
+			if ( url == null) {
+				error("unknown value for 'version' %s, 'version' can be a valid OSGi version, 'file', or 'url'", attrs);
+				return;
+			}
+			try (ToolManager toolmanager = new ToolManager(ws, this::progress)) {
+				toolmanager.install(url, attrs, options.force());
+				getInfo(toolmanager);
+			}
+		}
+		getInfo(ws);
+	}
+
+	private String getUrl(String key, Attrs attrs) {
+		String version = attrs.getVersion();
+		if (Verifier.isVersion(version)) {
+			return "https://github.com/bndtools/workspace.tool." + key + "/archive/refs/tags/" + version + ".zip";
```

Comment:
Is this a collection of new repos? `bndtools/workspace.tool.?`. Who will be maintaining these repos?

Reply:
You already maintain this for the workspace, so I thought of making a Github action?